EIPS/eip-EVM+.md

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
```suggestion
---
eip: 7543
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.